tests/json/microjson/valid/metadata-multilevel.json

```diff
@@ -0,0 +1,173 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        "subtype": "Rectangle",
+        "coordinates": [
+          [
+            [
+              0.0,
+              0.0
+            ],
+            [
+              0.0,
+              50.0
+            ],
+            [
+              50.0,
+              50.0
+            ],
+            [
+              50.0,
+              0.0
+            ],
+            [
+              0.0,
+              0.0
+            ]
+          ]
+        ]
+      },
+      "properties": {
+        "string": {
+          "well": "A1"
+        },
+        "numeric": {
+          "cellCount": 5
+        },
+        "multi-numeric": {
+          "ratioInfectivity": [
+            [
+              0.1,
+              0.2,
+              0.3,
+              0.4,
+              0.5
+            ],
+            [
+              0.2,
+              0.3,
+              0.4,
+              0.5,
+              0.6
+            ]
+          ]
+        }
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        "subtype": "Rectangle",
+        "coordinates": [
+          [
+            [
+              50.0,
+              0.0
+            ],
+            [
+              50.0,
+              50.0
+            ],
+            [
+              100.0,
+              50.0
+            ],
+            [
+              100.0,
+              0.0
+            ],
+            [
+              50.0,
+              0.0
+            ]
+          ]
+        ]
+      },
+      "properties": {
+        "string": {
+          "well": "A2"
+        },
+        "numeric": {
+          "cellCount": 10
+        },
+        "multi-numeric": {
```

Comment:
Examples for "multi-numeric" should'nt be list of floating numbers.

Reply:
In the example it is a ratio. Why is this inappropriate?

Reply:
"numeric" kind of inappropiate to define properties. I see this category handles both type of variable which are either numeric or floating number. Maybe other generalized term can be better suited to describe it